packages/core/src/utilities/background-color/background-color.scss

```diff
@@ -177,6 +177,10 @@
   background-color: $color-white !important;
 }
 
+.ds-u-fill--black {
+  background-color: $color-black !important;
+}
+
```

Comment:
Adding fill black as a bg color option as we have both white text and background options - this made sense to add